docs/deployment/batch_inference.ipynb

```diff
@@ -447,20 +454,25 @@
     "\n",
     "batch_run = project.run_function(\n",
     "    batch_inference,\n",
-    "    inputs={\"dataset\": prediction_set_path, \"sample_set\": training_set_path},\n",
+    "    inputs={\n",
+    "        \"dataset\": prediction_set_path, \n",
+    "        \"model_endpoint_sample_set\": training_set_path, \n",
+    "        \"model_path\": model_artifact.uri\n",
+    "            },\n",
     "    params={\n",
-    "        \"model\": model_artifact.uri,\n",
     "        \"label_columns\": \"label\",\n",
     "        \"perform_drift_analysis\": True,\n",
-    "    },\n",
+    "        \"trigger_monitoring_job\": True,\n",
+    "    }\n",
     ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this case, instead of just prediction, you will get drift analysis. The drift table plot that compares the drift between the training data and prediction data per feature:"
+    "In this case, instead of just prediction, you will get drift analysis. Note that if no label column has been provided, the job function will try to retrieve the label columns from the logged model artifact. If not defined in the model as well, the label columns will be generated with the following format `predicted_label_{i}` where `i` is an incremental number. \n",
```

Comment:
"In this case, instead of just prediction, you get drift analysis. If no label column was provided, the job function tries to retrieve the label columns from the logged model artifact. If also not defined in the model, the label columns are generated with the following format `predicted_label_{i}` where `i` is an incremental number. \n",